docs/self-managed/console-deployment/overview.md

```diff
@@ -0,0 +1,13 @@
+---
+id: overview
+title: "Console (Self-Managed)"
+sidebar_label: "Overview"
+description: ""
+---
+
+:::note
+Camunda Console Self-Managed is an [Alpha release](/reference/alpha-features.md). It is in an early state for you to test and participate in development by sharing your feedback.
+:::
+
+The Camunda Console Self-Managed offers a centralized overview for Camunda 8 installations, designed to enhance operational efficiency and oversight within enterprise environments. This console provides key insights into automation cluster deployments, process orchestration usage, and streamlining usage tracking.
+The Camunda Console Self-Managed provided in a container image. Refer to the [Installation Guide](../../self-managed/platform-deployment/overview.md) for details on how to install this component.
```

Comment:
"The Camunda Console Self-Managed provided in a container image" -- what do you mean by this? I think it is just a typo situation.